content/SCALETutorials/Instances/_index.md

```diff
@@ -19,554 +19,657 @@ keywords:
 
-<!-- nb: this is the content from the removed /scaletutorials/virtualization/addmanagevmdevicesscale/ -->
-<!-- preserved here in case any of it is useful when this tutorial is rewritten for Instances. -->
-<!--
-## Managing Devices
+   a. Enter a name for the virtual machine.
 
-After creating a VM, the next step is to add virtual devices for that VM.
-Using the Create Virtual Machine wizard configures at least one disk, NIC, and the display as part of the process.
-To add devices, from the **Virtual Machines** screen, click anywhere on a VM entry to expand it and show the options for the VM.
+      {{< include file="/static/includes/InstanceNameRequirements.md" >}}
 
-Click <i class="material-icons" aria-hidden="true" title="Devices">device_hub</i> **Devices** to open the **Devices** screen for the VM.
-From this screen, you can edit, add, or delete devices.
-Click the <span class="material-icons">more_vert</span> icon at the right of each listed device to see device options.
+   b. Select **VM** as the **Virtualization Method**.
 
-{{< trueimage src="/images/SCALE/Virtualization/VirtualMachinesScreenwithVMDetails.png" alt="VM Options" id="VM Options" >}}
+   c. Select one of the available **VM Image Options**: **Use a Linux Image** or **Upload ISO, import a zvol or use another volume**.
 
-The devices for the VM display as a list.
+      - Selecting **Use a Linux Image**:
 
-Device notes:
+         Select this option to create the VM using a Linux image from [linuxcontainers.org](https://linuxcontainers.org/).
 
-* A virtual machine attempts to boot from devices according to the **Device Order**, starting with **1000**, then ascending.
-* A **CD-ROM** device allows booting a VM from a CD-ROM image like an installation CD.
-  The CD image must be available in the system storage.
-* With a **Display** device, remote clients can connect to VM display sessions using a SPICE client, or by installing a 3rd party remote desktop server inside your VM.
-  SPICE clients are available from the [SPICE Protocol site](https://www.spice-space.org/).
+         a. Click **Browse Catalog** to open the **Select Image** screen.
 
-Before adding, editing, or deleting a VM device, stop the VM if it is running.
-Click the **State** toggle to stop or restart a VM, or use the **Stop** and **Restart** buttons.
+         {{< trueimage src="/images/SCALE/Virtualization/SelectImage.png" alt="Select Image Screen" id="Select Image Screen" >}}
 
-### Editing a Device
+         b. Search or browse to choose an available image.
 
-Select **Edit** to open the **Edit Device** screen.
-You can change the type of virtual hard disk, the storage volume to use, or change the device boot order.
+         c. Click **Select** in the row for your desired image.
 
-To edit a VM device:
+      - Selecting **Upload ISO, import a zvol or use another volume**:
```

Comment:
Same comment above, combining line 295 and 297 into one sentence, for example:

 "  - Select **Upload ISO, import a zvol or use another volume** to create the VM using an <file>.iso</file> image, import a zvol from a previously installed vm, or use and existing instance volume.
   Use one of the available options to select or create a volume:"

This should make this section a little more compact and the fewer bullet points helps with the flow.